File: src/product-service/models/order_result.go
package models

import uuid "github.com/satori/go.uuid"

type OrderResult struct {
	ID               int               ` json:"order_id" gorm:"primaryKey"`
	CustomerID       uuid.UUID         `json:"customer_id"`
	TotalAmount      float32           `json:"total_amount"`
	OrderResultItems []OrderResultItem `json:"order_result_items"`
}

File: src/ProductManagement/ProductManagement.Models/OrderResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProductManagement.Models
{
    public class OrderResult
    {
        public long Id { get; set; }
        public Guid CustomerId { get; set; }
        public decimal TotalAmount { get; set; }
        public ICollection<OrderResultItem> OrderResultItems { get; set; }
    }
}
